packages/app/src/systems/Welcome/components/WelcomePage.test.tsx

```diff
@@ -0,0 +1,20 @@
+import "cross-fetch/polyfill";
+import { renderWithRouter } from "@fuels-ui/test-utils";
+
+import { checkFirstLoad, createWallet } from "../__tests__/tests";
+
+import App from "~/App";
+
+describe("WalletPage", () => {
+  it("when first loaded, any route will redirect user to Create Wallet page", async () => {
+    await renderWithRouter(<App />, { route: "/pool/list" });
+    await checkFirstLoad();
+  });
+
+  it("should do create wallet flow", async () => {
+    await renderWithRouter(<App />, {
+      route: "/welcome/create-wallet",
+    });
+    await createWallet();
```

Comment:
I think all these functions that abstracts `tests` should start with `expect` or `test`. I normally don't see this type of abstraction and all the code is inside the test.

For example:  `testCreateWallet()` or `expectCreateWalletToBeValid()` this names are more readable and give more context of what  `createWallet` does.

Reply:
> I think all these functions that abstract tests should start with expect or test. I normally don't see this type of abstraction and all the code is inside the test.

This is the point for me as well